packages/rocketchat-lib/server/methods/sendInvitationSMS.js

```diff
@@ -3,8 +3,10 @@ import { TAPi18n } from 'meteor/tap:i18n';
 import { check } from 'meteor/check';
 import _ from 'underscore';
 
+const supportedLanguages = ['en'];
```

Comment:
I see now that this is likely why my test failed.  Will this be a configurable, or only just hardcoded?

Reply:
I also might think that this configuration item should be global to all types of invites.

Reply:
You need to add 'es' to this list, after you add the invite strings in spanish to the i18n package. Sure, we'll make it global when we handle the email invite.
This thing is not configurable via admin because you need to add strings to the i18n programmatically.

Reply:
Yeah, even after adding to the list and the i18n file for spanish it's not working yet.

Reply:
I'll check